app/models/user.rb

```diff
@@ -40,6 +40,15 @@ def self.user_info(parameter)
     end
   end
 
+  def user_tickets
+    bookings.order(id: :desc).decorate
+  end
+
+  def user_tickets_for_event(event_id)
+    bookings.where(
+      event_id: event_id).order(id: :desc).decorate
```

Comment:
Noted jay but rubocop settings does not allow characters longer than 80, so I had to break it.